application/src/main/java/org/opentripplanner/graph_builder/module/osm/ParkingProcessor.java

```diff
@@ -395,32 +398,29 @@ VehicleParking createVehicleParkingObjectFromOsmEntity(
       .build();
   }
 
-  private I18NString nameParkAndRideEntity(OsmWithTags osmWithTags) {
+  private I18NString nameParkAndRideEntity(OsmEntity osmEntity) {
     // If there is an explicit name user that. The explicit name is used so that tag-based
```

Comment:
```suggestion
    // If there is an explicit name use that. The explicit name is used so that tag-based
```

Not your fault, but please apply this typo fix.